### PR TITLE
Assignment1 submission

### PR DESCRIPTION
**Pull request by tomclark-mtcsw**

### CODE REVIEW

**Comment by GavenElder** [comment]
I will review this.

**Comment by GavenElder** [comment]
My PR for Tom's code:
Overall great work, I didn't find anything wrong with this code and worked flawlessly. 
I'm not sure if it makes a difference, but you could changed your pull title from "Assignment1 submission" to "Assignment1" (remove "submission").

    Pull Request title includes "Assignment 1" in the title ❌
    Pull Request targets Assignment 1 branch ✔
    Issue 1: Application no longer crashes ✔
    Issue 2: Unit test properly passes ✔

Extra Credit

    Issue 3: Feature request implemented ✔
    Issue 3: Feature request unit tested ✔

**Review by mmwoodfo — COMMENTED** [review]
👍🏻

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 1]
```diff
@@ -72,6 +73,28 @@ public void GetFilePath_ReturnsFileThatExists()
...
+                // Arrange
+                GenerateQuestionsFile(filePath, 10);
+
+                // Act
+                Question[] questions1 = Program.LoadQuestions(filePath);
+                Question[] questions2 = Program.LoadQuestions(filePath);
+                bool arraysAreEqual = questions1.SequenceEqual(questions2);
+
+                // Assert 
+                Assert.IsFalse(arraysAreEqual);
```
Comment (mmwoodfo) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:91]: This works totally fine, but you can also use `CollectionAssert` to one line it. Additionally, you can use the methods in the `CollectionAssert` class to check that the two arrays have the same elements, regardless of order. Just to prove that the randomization didn't remove or manipulate the elements in any way other than order.

Comment (mmwoodfo) [reply PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:91]: Additionally, in the event that the randomization manages to randomize the questions in the same order, this test may fail. A way around that is to simply increase your odds by shuffling a few more arrays and doing a comparison against 5 or so arrays to make sure at least two of them differ from each other.

But that's pretty unlikely

**Pull request closed without merging**